tpu/bootstrap.sh

```diff
@@ -53,6 +53,9 @@ while bash cluster-run.sh ls |grep 'exited with return code'; do
   sleep 10
 done
 
+# TODO: Remove the version pinning after tpu 1vm images are up-to-date.
+bash cluster-run.sh pip3 install tf-models-nightly tensorflow-text-nightly==2.10.0.dev20220718
```

Comment:
Do we want to freeze the tf-nightly version or tf model nightly version as well? (since the tf-text version is fixed here)?

Reply:
I have the feeling that tf-models automatically picks the same version as tensorflow-text-nightly. In that case pinning tf-models-nightly is not necessary.